python313Packages.aioshelly: 13.15.0 -> 13.17.0

fabaff opened the pull request:
Changelog: https://github.com/home-assistant-libs/aioshelly/releases/tag/13.17.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda reviewed (commented):
The commit message should say `13.15.0 -> 13.17.0`.
Add `bleak-retry-connector` to dependencies.

fabaff committed:
python313Packages.aioshelly: 13.15.0 -> 13.17.0

Changelog: https://github.com/home-assistant-libs/aioshelly/releases/tag/13.17.0

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 459896`
Commit: `791f5a8fc984a2527dffc262f71a6f369bdc93fb`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.tilt_ble</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 53 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.acaia</li>
    <li>home-assistant-component-tests.airthings_ble</li>
    <li>home-assistant-component-tests.aranet</li>
    <li>home-assistant-component-tests.bluemaestro</li>
    <li>home-assistant-component-tests.bluetooth_adapters</li>
    <li>home-assistant-component-tests.bluetooth_le_tracker</li>
    <li>home-assistant-component-tests.bthome</li>
    <li>home-assistant-component-tests.dormakaba_dkey</li>
    <li>home-assistant-component-tests.eq3btsmart</li>
    <li>home-assistant-component-tests.eufylife_ble</li>
    <li>home-assistant-component-tests.fjaraskupan</li>
    <li>home-assistant-component-tests.gardena_bluetooth</li>
    <li>home-assistant-component-tests.govee_ble</li>
    <li>home-assistant-component-tests.homekit_controller</li>
    <li>home-assistant-component-tests.husqvarna_automower_ble</li>
    <li>home-assistant-component-tests.ibeacon</li>
    <li>home-assistant-component-tests.idasen_desk</li>
    <li>home-assistant-component-tests.improv_ble</li>
    <li>home-assistant-component-tests.inkbird</li>
    <li>home-assistant-component-tests.iron_os</li>
    <li>home-assistant-component-tests.kegtron</li>
    <li>home-assistant-component-tests.keymitt_ble</li>
    <li>home-assistant-component-tests.kulersky</li>
    <li>home-assistant-component-tests.lamarzocco</li>
    <li>home-assistant-component-tests.ld2410_ble</li>
    <li>home-assistant-component-tests.leaone</li>
    <li>home-assistant-component-tests.led_ble</li>
    <li>home-assistant-component-tests.medcom_ble</li>
    <li>home-assistant-component-tests.melnor</li>
    <li>home-assistant-component-tests.moat</li>
    <li>home-assistant-component-tests.mopeka</li>
    <li>home-assistant-component-tests.motionblinds_ble</li>
    <li>home-assistant-component-tests.oralb</li>
    <li>home-assistant-component-tests.private_ble_device</li>
    <li>home-assistant-component-tests.probe_plus</li>
    <li>home-assistant-component-tests.qingping</li>
    <li>home-assistant-component-tests.rapt_ble</li>
    <li>home-assistant-component-tests.ruuvitag_ble</li>
    <li>home-assistant-component-tests.sensirion_ble</li>
    <li>home-assistant-component-tests.sensorpro</li>
    <li>home-assistant-component-tests.sensorpush</li>
    <li>home-assistant-component-tests.shelly</li>
    <li>home-assistant-component-tests.snooz</li>
    <li>home-assistant-component-tests.switchbot</li>
    <li>home-assistant-component-tests.thermobeacon</li>
    <li>home-assistant-component-tests.thermopro</li>
    <li>home-assistant-component-tests.xiaomi_ble</li>
    <li>home-assistant-component-tests.yalexs_ble</li>
    <li>home-assistant-component-tests.zeroconf</li>
    <li>python312Packages.aioshelly</li>
    <li>python312Packages.aioshelly.dist</li>
    <li>python313Packages.aioshelly</li>
    <li>python313Packages.aioshelly.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>home-assistant-component-tests.tilt_ble</summary>
<pre>              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.hass, handler_key, self._hass_config
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File &quot;/build/source/homeassistant/config_entries.py&quot;, line 3988, in _async_get_flow_handler
    await _load_integration(hass, domain, hass_config)
  File &quot;/build/source/homeassistant/config_entries.py&quot;, line 3974, in _load_integration
    raise data_entry_flow.UnknownHandler from err
homeassistant.data_entry_flow.UnknownHandler
=============================== warnings summary ===============================
../../nix/store/mkwsb9vjm5h3r1i51y81yq28acrd5gvx-python3.13-pytest-8.4.2/lib/python3.13/site-packages/_pytest/config/__init__.py:1474
  /nix/store/mkwsb9vjm5h3r1i51y81yq28acrd5gvx-python3.13-pytest-8.4.2/lib/python3.13/site-packages/_pytest/config/__init__.py:1474: PytestConfigWarning: Unknown config option: asyncio_debug
  
    self._warn_or_fail_if_strict(f&quot;Unknown config option: {key}\n&quot;)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR tests/components/tilt_ble/test_sensor.py::test_sensors - homeassistant.data_entry_flow.UnknownHandler
=================== 11 passed, 1 warning, 1 error in 13.26s ====================</pre>
</details>
</details>
